GNNGraphs/src/transform.jl

```diff
@@ -200,6 +210,20 @@ function remove_edges(g::GNNGraph{<:COO_T}, edges_to_remove::AbstractVector{<:In
              g.ndata, edata, g.gdata)
 end
 
+
+function remove_edges(g::GNNGraph{<:COO_T}, p = 0.5)
+    num_edges = g.num_edges
+    edges_to_remove = filter(_ -> rand() < p, 1:num_edges)        
+    g = remove_edges(g, edges_to_remove)
+    s, t = edge_index(g)
+    w = get_edge_weight(g)
+    edata = g.edata
+    return GNNGraph((s, t, w),
+             g.num_nodes, length(s), g.num_graphs,
+             g.graph_indicator,
+             g.ndata, edata, g.gdata)
```

Comment:
there is any reason for not doing simply
```suggestion
    return remove_edges(g, edges_to_remove)
```
?

Reply:
Yes, my bad